app/assets/stylesheets/modules/flash.scss

```diff
@@ -0,0 +1,16 @@
+@import "variables";
+
+.flash-container {
+  text-align: center;
+  left: 35%;
+  right: 35%;
+  background-color: rgba( $dark-red, .5)
```

Comment:
`.5` should be written with a leading zero as `0.5`<br>Expected 0 spaces between parentheses instead of 1<br>Declaration should be terminated by a semicolon